src/mlpack/tests/lsh_test.cpp

```diff
@@ -117,8 +117,10 @@ BOOST_AUTO_TEST_CASE(NumTablesTest)
   const string testSet = "iris_test.csv";
   arma::mat rdata;
   arma::mat qdata;
-  data::Load(trainSet, rdata, true);
-  data::Load(testSet, qdata, true);
+  if (!data::Load(trainSet, rdata, true)) 
+    FAIL("Unable to load data rainSe"); 
```

Comment:
```suggestion
    FAIL("Cannot load test dataset iris_train.csv! ");
```
Some issue with the script here?
It says ```rainSe```.

In this test case, they are loading the ```iris_train.csv``` and dataset ```iris_test.csv```. 
It would be better if the message included the name of the dataset.